File: matlab/transform_to_normal_data.m
%clear all; close all;
rng('default'); rng(263);
n = 30;
y0 = randn(n, 1);
x = sort(1./(1+exp(-4*(y0+1)))+(1./(1+exp(-y0))+2./(1+exp(-6*(y0-2)))))-1.2;

File: python/transform_to_normal_data.py
import numpy as np

np.random.seed(364)
n = 30
y0 = np.random.randn(n, )
x = np.sort(1/(1+np.exp(-4*(y0+1)))+(1/(1+np.exp(-y0))+2/(1+np.exp(-6*(y0-2)))))-1.2